helm/thingsboard/templates/node.yaml

```
@@ -123,6 +123,8 @@ spec:
           {{- end }}
           - name: DEFAULT_INACTIVITY_TIMEOUT
             value: {{ .Values.node.state.defaultInactivityTimeoutInSec | default 600 | quote }}
+          - name: DEFAULT_CHECK_INTERVAL
```

```suggestion
          - name: DEFAULT_STATE_CHECK_INTERVAL
```
?
https://github.com/thingsboard/thingsboard/blob/733dcea36ad315d3ee377195402793ce94ced8d4/application/src/main/resources/thingsboard.yml#L655

---

Done!